src/main/java/io/antmedia/console/rest/CommonRestService.java

```diff
@@ -261,17 +262,30 @@ public boolean sendUserInfo(String email, String firstname, String lastname, Str
 
 			httpPost.setConfig(requestConfig);
 
+			// if any of them is null, then it will throw IllegalArgumentException
+			// so, be sure that none of them is null
```

Comment:
Are you sure that this comment is true?

Reply:
it think comment wrong, maybe I should change it to "if args are null, set them to empty string"

Reply:
Yes, please

Reply:
It's done

Reply:
Thank you for the update